tools.mk

```diff
@@ -27,6 +27,13 @@
 REBAR ?= ./rebar
```

Comment:
This is simply the current one if `REBAR` is not set. Not sure rebar is in the path by default...